src/agenda/index.js

```diff
@@ -119,6 +115,8 @@ export default class AgendaView extends Component {
     };
 
     this.currentMonth = this.state.selectedDay.clone();
+    this.onHover = this.onHover.bind(this);
```

Comment:
What these changes has to do with fixing the rendering? I think you're mixing two tasks here...

Reply:
This adds mouse support so that the interactions can work on web. I stole this from [here](https://github.com/bkniffler/react-native-calendars/commit/7a0f986bbef1f84b0fd6901af6974c8bcd7436c2) and credited the author in the main PR comment.